frontend/app/types.d.ts

```diff
@@ -98,7 +98,7 @@ declare module 'i18next' {
       gcweb: typeof gcweb;
       'personal-information': typeof personalInformation;
       'update-phone-number': typeof updatePhoneNumber;
-      'view-letters': typeof viewLetters;
+      'letters': typeof Letters;
```

Comment:
To follow existing conventions:

```suggestion
      'letters': typeof letters;
```